pkg/detectors/saucelabs/saucelabs.go

```diff
@@ -91,3 +78,30 @@ func (s Scanner) Type() detectorspb.DetectorType {
 func (s Scanner) Description() string {
 	return "A service for cross browser testing, API keys can create and access tests from potentially sensitive internal websites"
 }
+
+func verifySauceLabKey(ctx context.Context, client *http.Client, userName, key string) (bool, error) {
+	req, err := http.NewRequestWithContext(ctx, "GET", "https://api.eu-central-1.saucelabs.com/team-management/v1/teams", nil)
```

Comment:
What about us-west and us-east?

Reply:
I thought about that and the solution I have on my mind is to search for domain in chunk, if found use that else used a fixed one. What do you think? Earlier we were only using one.
Also during testing I noticed that if we hit wrong domain with valid token it gives us `403` and if token is incorrect we get `401`.

Reply:
@rgmz can you check now? If current approach looks ok to you.

Reply:
That's probably fine. I did something similar [here](https://github.com/trufflesecurity/trufflehog/blob/31b4dc2fb7633124245f30b3eb8e1b35e031ee39/pkg/detectors/sumologickey/sumologickey.go#L67).

In theory, I think you're supposed to use [`CloudProvider`](https://github.com/trufflesecurity/trufflehog/blob/main/pkg/detectors/detectors.go#L83) / [`EndpointCustomizer`](https://github.com/trufflesecurity/trufflehog/blob/e5d138d824cac29d3e99d41ffcda9e2b7397d12d/pkg/detectors/detectors.go#L76) / [`EndpointSetter`](https://github.com/trufflesecurity/trufflehog/blob/e5d138d824cac29d3e99d41ffcda9e2b7397d12d/pkg/detectors/endpoint_customizer.go#L12); I have no clue how tf to actually use them.

Reply:
I think that is to provide users the functionality to pass their own custom endpoints for the application/software vs using the default cloud endpoint. Here we have only three fix endpoints to choose from. Correct me @mcastorina If I am wrong about the usage of EndpointCustomizer.